libs/locales/lib/es/translation.json

```diff
@@ -335,7 +335,7 @@
   "ai:For example: host-{{n}}": "Por ejemplo: host-{{n}}",
   "ai:Format?": "¿Formato?",
   "ai:Forwarding it could put your credentials and personal data at risk.": "Reenviarlo podría poner en riesgo sus credenciales y datos personales.",
-  "ai:Full image file: Provision with physical media": "Archivo de imagen completo: Provisión con soporte físico",
```

Comment:
how did you update these other languages ? Generally we just update the `en` ones and keep these for the i18n team

Reply:
I updated them manually.. should i remove the other ones?

Reply:
yes, lets remove the changes to the other ones

Reply:
Done